### DISCUSSION TITLE
Rendering react-pdf components in grapejs editor

### DISCUSSION THREAD

**Discussion by BashisthaSudeep** [GrapesJS/grapesjs#4694] [Q&A]
I'm trying to render the @react-pdf/renderer components in grapesjs editor. I've templates designed using @react-pdf/renderer components whose content can be edited and downloaded as a pdf document.

When rendering the react-pdf component, it renders an empty iframe inside grapesjs editor if I use PDFViewer component provided by react-pdf. If I don't use PDFViewer component I get the following error but the content is rendered. 

`next-dev.js?3515:24 Warning: <TEXT /> is using incorrect casing. Use PascalCase for React components, or lowercase for HTML elements.`

Can anyone who has done this point out what I'm missing?

**Comment by dohomi** [comment]
@BashisthaSudeep did you have any success with the integration?

**Reply by BashisthaSudeep** [reply]
Yes. I had to get rid of react-pdf and do everything from plain html.

**Reply by dohomi** [reply]
did you use something like pdfmake or how did you create the pdf out of html/css?

**Reply by BashisthaSudeep** [reply]
I'm now rendering them on the server side.
